expiringdict/__init__.py

```diff
@@ -110,22 +114,36 @@ def get(self, key, default=None, with_age=False):
     def items(self):
         """ Return a copy of the dictionary's list of (key, value) pairs. """
         r = []
-        for key in self:
-            try:
-                r.append((key, self[key]))
-            except KeyError:
-                pass
+        if sys.version_info >= (3, 5):
```

Comment:
I am not very fond of this solution for two reasons. Code duplication and the version check is performed every time when we call the method. How about we create `_safe_keys` alias at the class level:

```python
if sys.version_info >= (3, 5):
    _safe_keys = lambda self: list(self.keys())
else:
    _safe_keys = keys
```
And will use it as follows:
```
def items(self):
        """ Return a copy of the dictionary's list of (key, value) pairs. """
        r = []
        for key in self._safe_keys():
            try:
                r.append((key, self[key]))
            except KeyError:
                pass
        return r
```
In this case version check is performed only once when the class is imported and code duplication is avoided.